helpers/renderingInfoScript.js

```diff
@@ -40,10 +40,12 @@ function getScript(
 
     author.innerHTML = "";
     if(image.credit) {
-      if(image.credit.link.url && image.credit.link.isValid) {
-        author.innerHTML = "<a class='q-imageslider-description__author__url' href='" + image.credit.link.url + "' target='blank' rel='noopener noreferrer'>" + image.credit.text + "</a>";
-      } else if(image.credit.text) {
-        author.innerHTML = image.credit.text;
+      if(image.credit.text) {
```

Comment:
Couldn't you add this if-condition to the condition of the previous line

Reply:
I could. You're right. Shame on lazy me